workflow/scripts/predict.py

```diff
@@ -190,26 +198,48 @@ def main():
             "Expression",
             "PromoterActivityQuantile",
             "isExpressed",
-            "DHS.RPKM.quantile.TSS1Kb",
+            args.default_accessibility_feature + ".RPKM.quantile.TSS1Kb",
         ],
     ]
-    genes.columns = [
-        "chr",
-        "TargetGene",
-        "TargetGeneTSS",
-        "TargetGeneExpression",
-        "TargetGenePromoterActivityQuantile",
-        "TargetGeneIsExpressed",
-        "normalized_dhs",
-    ]
+    if args.default_accessibility_feature == "ATAC":
+        genes.columns = [
+            "chr",
+            "TargetGene",
+            "TargetGeneTSS",
+            "TargetGeneExpression",
+            "TargetGenePromoterActivityQuantile",
+            "TargetGeneIsExpressed",
```

Comment:
This part looks like its shared with DHS. We can set this outside the for loop to prevent repetition?